.tool-versions

```diff
@@ -0,0 +1,3 @@
+elixir 1.14.3-otp-25
+nodejs 18.13.0
+erlang 25.3
```

Comment:
why do we need this file?

Reply:
to use asdf to install the required versions to run realtime. makes it easier to onboard people to the project if need be.

with this and asdf we just need to do `asdf install` to have the dev environment setup

Reply:
sounds good 👍